tests/test_cantools.py

```diff
@@ -24,7 +24,7 @@ def test_vehicle(self):
 
     def test_motohawk(self):
         f = cantools.db.File()
-        filename = os.path.join('tests', 'files', 'motohawk.dbc')
+        filename = os.path.join('files', 'motohawk.dbc')
```

Comment:
The tests on Travis fails (https://travis-ci.org/eerimoq/cantools/jobs/220106672), partly due to this path change. Please fix.

You can run the tests locally with ``python setup.py test`` and ``python3 setup.py test`` in the repository root folder.